src/ui/button/button.jsx

```diff
@@ -21,85 +21,87 @@ export function validateButtonProps(props : ButtonProps) {
 }
 
 type LogoProps = {|
-    style : ButtonStyle
-|};
-
-type PPCreditLogoProps = {|
     style : ButtonStyle,
     locale : LocaleType
 |};
 
+// type PPCreditLogoProps = {|
```

Comment:
Can we delete all the commented out code? Seems like its not needed.

Reply:
done